tests/qir/test_pytket_qir_rangepredicate.ll

```diff
@@ -11,32 +11,26 @@ define void @main() #0 {
 entry:
```

Comment:
This file is part of the tests, it is just used to compared the generated qir to what we are expecting